website/blog/2025-12-16-aks-config-scheduler-profiles-preview/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Announcing Azure Configurable Scheduler Profiles (Preview): Optimize resource ROI with fine grained scheduling strategies"
+description: "Avoid scheduling inefficiencies and improve GPU utilization with AKS Configurable Scheduler Profiles"
+date: 2025-12-16
+authors: 
+   - colin-mixon
+tags:
+  - ai
+  - performance
+  - scheduler
+  - best-practices
+---
+
+Thoughtful scheduling strategies can resolve pervasive challenges across web-distributed workloads and AI workloads like resiliency and resource utilization. But the default scheduler was primarily designed for general-purpose workloads and out-of-box pod scheduling that could be restrictive if you needed more fine-grain control since it applies a set of criteria in a fixed priority order. The scheduler selects the optimal node for newly created pod(s) based on several criteria, including (but not limited to):
+
+1. Resource requirements (CPU, memory)
+2. Node affinity/anti-affinity
+3. Pod affinity/anti-affinity
+4. Taints and tolerations
+
+Out of the available nodes, the scheduler then filters out nodes that don't meet the requirements to identify the node that is most optimal for the pod(s). Today, the AKS default scheduler lacks the flexibility for users to change which criteria should be prioritized, and ignored, in the scheduling cycle on a per workload basis. This means the default scheduling criteria, and their fixed priority order, are not suitable for workloads that demand co-locating pods with their persistent volumes for increased data locality, optimizing GPU utilization for machine learning, or enforcing strict zone-level distribution for microservices. This rigidity often forces users to either accept suboptimal placement or manage a separate custom scheduler, both of which increase operational complexity.
```

Comment:
You can start with how the AKS default scheduler is different if any from the Kuberentes scheduler upstream defined.